grouper/fe/forms.py

```diff
@@ -264,3 +265,12 @@ class UserTokenForm(Form):
     name = StringField("Token name", [
         validators.DataRequired(),
     ])
+
+
+class UserPasswordForm(Form):
+    name = StringField("Password name", [
```

Comment:
We have historically had a single password hash for a given user.  It would be better from a security standpoint to let the user create multiple hashes, one for each service, but in practice the only service that still uses this is TACACS.  So I'm not sure it's worth spending too much time or effort on.